lib/message.rb

```diff
@@ -0,0 +1,18 @@
+require 'twilio-ruby'
+
+class Message
+  SUCCESS_MSG = "Thank you! Your order was placed and will be delivered before 18:52"
+
+  def initialize
+    account_sid = ENV['TWILIO_ACCOUNT_SID']
+    auth_token = ENV['TWILIO_AUTH_TOKEN']
+    @client = Twilio::REST::Client.new(account_sid, auth_token)
```

Comment:
By creating a new instance of the twilio client inside of your initialize method like this you're setting up a hard dependency between your two objects and making it very difficult to isolate your unit tests. The biggest problem with that in this particular case is that you wont be able to test this class without sending yourself a text message every time you run rspec 🙂 

You can start to get around this by making use of dependency injection:

```ruby
def initialize(client = Twilio::REST::Client.new(account_sid, auth_token))
  @client = client
end
```